src/server/announcements.ts

```diff
@@ -1,9 +1,17 @@
 import { nanoid } from 'nanoid'
 import ActivityPubSystem, { DEFAULT_PUBLIC_KEY_FIELD } from './apsystem'
 import { generateKeypair } from 'http-signed-fetch'
-import { APOrderedCollection } from 'activitypub-types'
+import { APOrderedCollection, APActor } from 'activitypub-types'
 import { ActorStore } from './store/ActorStore'
 
+type APActorNonStandard = APActor & {
+  publicKey: {
+    id: string
+    owner: string
+    publicKeyPem: string
+  }
+}
+
```

Comment:
this should probably be moved elsewhere but i'm not sure where. maybe src/schemas.ts? either way it can be moved later if it's used elsewhere

Reply:
Yeah mind moving it to schemas? Also maybe we could PR the field into the module? They accepted my orderedItems one

Reply:
i'm looking into doing the PR but I can't find the spec that defines keypair in APActor